git-cliff-core/Cargo.toml

```diff
@@ -79,6 +79,7 @@ features = ["debug-embed", "compression"]
 
 [dev-dependencies]
 pretty_assertions = "1.4.0"
+expect-test = "1.5.0"
```

Comment:
I added this dependency because it makes snapshot testing very easy.
you just run `env UPDATE_EXPECT=1 cargo t` and it will update your strings to make tests pass.

I was a bit tired of adding and removing spaces, but if you don't want to add a library (dev-dependency), I'll do it 👍

Reply:
Sounds fair, can you document this behavior in `CONTRIBUTING.md`?

Reply:
done in [0e81dcf](https://github.com/orhun/git-cliff/pull/613/commits/0e81dcf243ad0aac8af45ec46920f540d4765988)